### ISSUE TITLE
Silent failures on double-write errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [j-woz/exm-issues#323]
```
At some point ADLB started letting double write errors 

We really need to have these checks enabled and throwing errors by default 
(like array bounds checking in other languages for example): otherwise users 
get no feedback on erroneous code.

I checked in 312 as a test for this.
```

Original issue reported on code.google.com by `tim.g.ar...@gmail.com` on 22 Oct 2012 at 11:18
- Blocked on: #457
- Merged into: #457
